src/backend/web/routes/opml.js

```diff
@@ -1,25 +1,33 @@
 const express = require('express');
 const opml = require('opml-generator');
+const { getFeeds, getFeed } = require('../../utils/storage');
+const { logger } = require('../../utils/logger');
 
 const router = express.Router();
 
 const header = {
   title: 'OPML Feeds',
-  dateCreated: new Date(2020, 2, 9),
-  ownerName: 'Jordan',
+  dateCreated: new Date(),
+  ownerName: 'Telescope',
 };
 
-const outlines = [
-  {
-    text: 'txt',
-    title: 'My Open Source Experience',
-    type: 'rss',
-    xmlUrl: 'https://jrdnlxopensource.blogspot.com/feeds/posts/default/-/open-source',
-    htmlUrl: 'https://jrdnlxopensource.blogspot.com/',
-  },
-];
+router.get('/', async (req, res) => {
+  let feeds;
+  try {
+    feeds = await Promise.all((await getFeeds()).map(feedId => getFeed(feedId)));
```

Comment:
I've slowly been trying to move us away from calling directly into Redis, and use helper classes instead.  The value of this is that it means we can clean-up data, or change types as necessary before we try to use raw data from the database.

As such, can you change your final `.map()` to use the `Feed` class from `src/backend/data/feed`?  It would look like this:

```js
feeds = await Promise.all((await getFeeds()).map(id => Feed.byId(id)));
```

You could shorten down a bit more even:

```js
feeds = await Promise.all((await getFeeds()).map(Feed.byId));
```

Reply:
Great idea; done in https://github.com/Seneca-CDOT/telescope/pull/594/commits/8ecc98f28c14b82023cc988fd1e0999a86f37853